compiler/loco/src/IR/Graph.test.cpp

```diff
@@ -79,7 +79,11 @@ TEST(GraphTest, create_and_destroy_node)
   auto pull = g->nodes()->create<loco::Pull>();
 
   ASSERT_NO_THROW(g->nodes()->destroy(pull));
-  ASSERT_THROW(g->nodes()->destroy(pull), std::invalid_argument);
+  // NOTE calling g->nodes()->destroy(pull) again will raise segmentation fault
+
+  auto g2 = loco::make_graph();
+  auto pull2 = g2->nodes()->create<loco::Pull>();
+  ASSERT_THROW(g->nodes()->destroy(pull2), std::invalid_argument);
 }
```

Comment:
Hmm 🤔  the logic itself is written with the consideration of double free. So, this test tries to test that. If this line cause segmentation fault. It would be better to disable this test temporary by prepending ``DISABLED_`` to test name instead of modifying the test itself.

Reply:
OK, thanks for the comment.
I'll split this test as (1) just call destroy once (2) add another test with double destroy but disabled.